frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx

```diff
@@ -20,11 +20,31 @@ const StyledMissionView = styled.div`
     align-self: stretch;
     border-top: 1px solid ${tokens.colors.ui.background__medium.hex};
 `
+const StyledTitleAndButton = styled.div`
+    display: flex;
+    flex-directon: row;
+    justify-content: space-between;
+    align-items: center;
+`
+
+const StyledDialog = styled(Dialog)`
```

Comment:
import { StyledDialog } from 'components/Styles/StyledComponents'

Can we reuse this instead of redefining it?